tests/system/filesystem_integrity/README.md

```diff
@@ -0,0 +1,70 @@
+# Filesystem integrity test
```

Comment:
Have you tried to run the test according to this documentation?

Reply:
:+1: